examples/performance-example/app/src/main/java/com/example/bugsnag/performance/MainActivity.kt

```diff
@@ -16,20 +29,45 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.drawWithContent
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
 import com.bugsnag.android.performance.BugsnagPerformance
 import com.bugsnag.android.performance.SpanOptions
-import com.bugsnag.android.performance.compose.MeasuredComposable
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 
+
 class MainActivity : AppCompatActivity() {
     private val network = ExampleNetworkCalls(this)
 
+    @OptIn(ExperimentalMaterial3Api::class)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        ActivityView(this).edgeView()
         setContent {
-            MeasuredComposable(name = "MainActivity") {
-                AllButtons(network = network)
+            Scaffold (
+                topBar = {
+                    TopAppBar(
+                        colors = TopAppBarDefaults.topAppBarColors(
+                            containerColor = colorResource(R.color.purple_500),
+                            titleContentColor = colorResource(R.color.white),
+
+                        ),
+                        title = {
+                            Text(
+                                text = resources.getString(com.example.bugsnag.performance.R.string.app_name),
+
+                            )
+                        },
+                    )
+                },
+                modifier = Modifier.windowInsetsPadding(WindowInsets.systemBars),
+
+                ) { contentPadding ->
+                Column(
+                    Modifier.padding(contentPadding)
+                ) {
+                    AllButtons(network = network)
+                }
```

Comment:
Given that `AllButtons` is a column, I think that the `contentPadding` modifier should rather be passed down here.